index.html

```diff
@@ -2044,6 +2146,76 @@ <h2 id="links">Links &amp; Suggested Reading</h2>
 
       <h2 id="changelog">Change Log</h2>
 
+      <p id="1.8.0">
+        <b class="header">1.8.0</b> &mdash; <small><i>Jan 23, 2015</i></small> &mdash; <a href="https://github.com/jashkenas/underscore/compare/1.7.0...1.8.0">Diff</a> &mdash; <a href="https://cdn.rawgit.com/jashkenas/underscore/1.8.0/index.html">Docs</a><br />
+        <ul>
+          <li>
+            Added <tt>_.keysIn</tt> which returns the enumerable properties of an object.
+          </li>
+          <li>
+            Reverted change where <tt>_.extend</tt> only iterates own properties, it now uses <tt>_.keysIn</tt>. Added <tt>_.assign</tt> that aligns closely with ECMA6 <tt>Object.assign</tt> and iterates over own properties.
+          </li>
+
+          <li>
+            Added <tt>_.create</tt> helper for simpler inheritance patterns.
+          </li>
+
+          <li>
+            Added extremely handy <tt>_.transform</tt> method as an alternative to <tt>_.reduce</tt> in many cases.
+          </li
+>
+          <li>
+            Added <tt>_.isError</tt> which determines if an object is derived from the <tt>Error</tt> object.
+          </li>
+
+          <li>
+            Added <tt>_.unzip</tt> again for easier use in chaining.
+          </li>
+
+          <li>
+            <tt>_.result</tt> now takes an optional default value argument. If this argument is a function
+            it is called.
+          </li>
+
+          <li>
+            Added <tt>_.propertyOf</tt> wrapper as a flip of <tt>_.property</tt>.
+          </li>
+          <li>
+            <tt>_.property</tt> now returns <tt>undefined</tt> when passed <tt>null</tt> or <tt>undefined</tt>.
+          </li>
+
+          <li>
+            <tt>_.partial</tt> can now be used with constructors!
+          </li>
+
+          <li>
+            <tt>_.uniq</tt> now uses sorted iteration when provided <tt>sorted=true</tt> and given an <tt>iteratee</tt> argument.
+          </li>
+
+          <li>
+            Aliased <tt>_.contains</tt> as <tt>_.includes</tt> and added support for finding <tt>NaN</tt>.
+          </li>
+          <li>
+            <tt>_.contains</tt> can be provided an optional start index.
+          </li>
+          <li>
+            <tt>_.indexOf</tt> and <tt>_.lastIndexOf</tt> now support search for <tt>NaN</tt>.
+          </li>
+
+          <li>
+            <tt>_.iteratee</tt> no longer takes the wrapped functions expected argument length.
```

Comment:
`wrapped functions` -> `wrapped functions'` or `wrapped function's`.

This could also be worded a bit better. Maybe "`_.iteratee` no longer exposes the internal `argCount` parameter for [...]"